internal/receiver/smartagentreceiver/testdata/extension_config.yaml

```diff
@@ -0,0 +1,34 @@
+extensions:
+  smartagent/partial_settings:
+    bundleDir: /opt/
+    collectd:
+      timeout: 10
+      readThreads: 1
+      writeThreads: 4
+      writeQueueLimitHigh: 5
+      configDir: /etc/
+  smartagent/extra:
+    bundleDir: /opt/
+    collectd:
+      timeout: 10
+      readThreads: 1
+      writeThreads: 4
+      writeQueueLimitHigh: 5
+      configDir: /etc/
+
+receivers:
+  examplereceiver:
+
+processors:
+  exampleprocessor:
+
+exporters:
+  exampleexporter:
+
+service:
+  extensions: [smartagent/partial_settings]
```

Comment:
missing the extra one?

Reply:
oh, guessing this mechanism doesn't propagate to the test host...  disregard

Reply:
yeah, was about say but updated regardless.